packages/ckeditor5-paste-from-office/src/normalizers/googledocsnormalizer.js

```diff
@@ -45,8 +45,11 @@ export default class GoogleDocsNormalizer {
 	 */
 	execute( data ) {
 		const writer = new UpcastWriter( this.document );
+		const { body } = data._parsedData;
```

Comment:
Maybe we could rename the `body` variable to `documentFragment` inside this method? All these names here are quite hard to distinguish like `data`, `data._parsedData`, `body`, `data.content` :smile:

Reply:
Good point. I've renamed `body` to `documentFragment` and `bodyString` to `documentFragmentString` to be consistent.

Reply:
It turned out that I got it wrong and I shouldn't change all these properties as it will create a breaking change.

Now, I just used the `documentFragment` variable name inside `execute()` method in both normalizers.